pages/submissions/[id]/index.tsx

```diff
@@ -34,9 +34,14 @@ const TaskListPage = ({
   const completedSteps = Object.keys(formAnswers);
   const person = residents[0];
 
-  const handleFinish = async (): Promise<void> => {
+  const handleFinish = async (values?: {
```

Comment:
Why is this optional? Won't we always need the approveEmail to finish a submission